hack/terraform-quickstart/genlogs.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+export IDENT=${IDENT:-${HOME}/.ssh/id_rsa}
+export WORKER_IPS=`terraform output -json worker_ips | jq -r '.value[]'`
+export MASTER_IPS=`terraform output -json master_ips | jq -r '.value[]'`
+export BOOTSTRAP_IP=`terraform output bootstrap_node_ip`
+export LOGS_DIR=${LOGS_DIR:-/tmp/logs}
+export REMOTE_PORT=${REMOTE_PORT:-22}
+export REMOTE_USER=${REMOTE_USER:-core}
+export IDENT=${IDENT:-${HOME}/.ssh/id_rsa}
```

Comment:
This is provided twice. Also, shouldn't ssh just work? We already ssh to the box in the terraform apply.

Reply:
fixed in 9939c6d8

Reply:
the identity needs to be passed to ssh... we could let ssh guess, but the other scripts do it this way. My guess as to why is to allow the ssh scripts to be run locally to a provisioned cloud vm not managed by terraform.